tools/nodejs/README.md

```diff
@@ -99,3 +99,14 @@ var stmt = con.prepare('select ?::INTEGER as fortytwo', function(err, stmt) {
   });
 });
 ```
+
+## Development
+
+Tests are located in `tools/nodejs/test`, these tests require Mocha to run.
+Mocha can be installed with `npm install mocha chai --save-dev`
```

Comment:
Mocha and chai are installed automatically as dev dependencies when you run `npm install`? Though you may want `--ignore-scripts` if you don't want to build the extension at the same time

Reply:
I also realized this just now, did not know the `--ignore-scripts` option though, I'll add it